src/TodoList.jsx

```diff
@@ -1,13 +1,17 @@
 import TodoListItem from "./TodoListItem";
-function TodoList({todoList}){
+function TodoList({todoList, onCompleteTodo}){
+    const filteredTodoList = todoList.filter((ele) => ele.isCompleted === false)
```

Comment:
I understand that `ele` is element but I'm not sure I would understand it if I was looking at this code the first time.